.mailmap

```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <zhanglei@trendytech.com.cn> <243290414@qq.com>
```

Comment:
I think you should swap this to make it similar to the entries made by other people.
`Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn>`

Also insert your line in the alphabetical order.